roles/openshift_health_checker/library/ocutil.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+# vim: expandtab:tabstop=4:shiftwidth=4
+"""
+  Interface to OpenShift oc command
+"""
+#
+#   Copyright 2017 Red Hat Inc.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+import shlex
+import subprocess
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+def main():
+    """ Handle args and execute OCUtil method """
+    module = AnsibleModule(
+        argument_spec=dict(
+            namespace=dict(type="str", required=True),
+            config_file=dict(type="str", required=True),
+            cmd=dict(type="str", required=True)
+        ),
+        supports_check_mode=True
+    )
+
+    config_file = module.params["config_file"]
+
+    cmd = " ".join(['oc --config', config_file, '-n', module.params["namespace"], module.params["cmd"]])
+    cmd_split = shlex.split(cmd)
+
+    failed = False
+
+    try:
+        cmd_result = subprocess.check_output(cmd_split)
+    except subprocess.CalledProcessError as err:
+        failed = True
+        cmd_result = err
+
+    module.exit_json(
+        changed=False,
+        failed=failed,
+        ansible_facts={
```

Comment:
This has some special meaning http://docs.ansible.com/ansible/common_return_values.html#ansible-facts

Why are we using it?

Won't it pollute the host vars?